docs/release_process.md

```diff
@@ -0,0 +1,25 @@
+# Release process
+
+## Frequency
+
+Open Service Mesh will target a release every quarter. These releases will be targeted in the first half of February, May, August, and November. These target release months are subject to change if a release is pushed. In other words, if a release misses February and is pushed out in March then the next release will target June instead of May.
+
+## Process for requesting a change
+
+If you would like to see a new feature added to Open Service Mesh, please [open an issue](https://github.com/openservicemesh/osm/issues). This issue will be discussed and, if approved, prioritized in the backlog.
+
+## Creating a new release
+
+When there is a new release being planned and developed, a [milestone](https://github.com/openservicemesh/osm/milestones) will be created. This new milestone will have issues added to it that will be prioritized from the [vFuture milestone](https://github.com/openservicemesh/osm/milestone/32). Once the release milestone is complete, a new release of Open Service Mesh will be created and this milestone will be closed. At that moment, the next release milestone will be created and items from vFuture will be added.
+
+## Backlog handling
```

Comment:
How are longer projects handled, ie: cert rotation? These are not yet designed, not yet estimated projects. Even when they are estimated larger projects often tend to slip past deadlines. Do we assign these to a release after they are complete? When they are near completion?

One thing I've seen work is to use an additional label, "release-blocker",  and then we can be more aggressive about adding items to releases.

Reply:
That's a good question and something I should include in this documentation. For longer projects they should likely have their own milestone that isn't tied to a release. They should be worked in parallel to the release(s) during that time but considered for "next release" when the feature milestone completion date lines up with a release milestone completion date.

Reply:
Otherwise folks will be having tasks in the  *in progress* column that are not attributed to a release.